internal/statepb/statepb.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package statepb contains code generated from proto/pipeline.proto, for
+// Librarian state and config files. This package is likely to be deleted
+// as part of https://github.com/googleapis/librarian/issues/326. No
+// handwritten code should be added to this package.
+package statepb
```

Comment:
since this file is only intended to contain package documentation, it is convention to name it to doc.go